build/release/release-procedure.adoc

```diff
@@ -0,0 +1,96 @@
+= Release Procedure
+
+== Projects and Artifacts
+Following sections describe what projects are part of the OptaPlanner release procedure and their resulting
+release artefacts.
+
+=== OptaPlanner
+
+* JAR files hosted on Maven Central
+* https://download.jboss.org/optaplanner/release/7.44.0.Final/optaplanner-distribution-7.44.0.Final.zip[Distribution.zip]
+available via a link on the optaplanner-website
+* https://docs.optaplanner.org/7.44.0.Final/optaplanner-docs/html_single/index.html[Documentation]
+available via a link on the optaplanner-website
+* XML Schema Definition files for both the https://www.optaplanner.org/xsd/solver/[Solver]
+and the https://www.optaplanner.org/xsd/benchmark/[Benchmark] available on the optaplanner-website
+
+=== OptaPlanner Quickstarts
+
+No build artifacts are required; as a result of the release, the development branch must be rebased onto
+the stable branch and must point to the released final version of OptaPlanner.
```

Comment:
I'd use a different typeface to show that `stable` and `development` are, in fact, actual branch names.